controller/controller.php

```diff
@@ -117,4 +124,32 @@ function contactMe() {
 }
 
 
+// Determine the posts list and page number to display them :
+function postsPager() {
+    
+    //obtain posts :
+    $postManager = new postManager();
+    $posts=$postManager->getList();
+
+    //Determine pages number to display 3 posts at a time
+    $posts=array_chunk($posts, 3);
+    $p_pages_nbr=array_key_last($posts)+1;
```

Comment:
camelCase is generally preferred to snake_case, but that's not a big deal. On the other hand, you should choose one case style and stick to it throughout your project.

Reply:
Ok, I will change my variables names to camelCase so. I prefer do the things as appropriate. :)